komoju-php/komoju-php/lib/komoju/KomojuApi.php

```diff
@@ -89,6 +94,7 @@ private function get($uri, $asArray = false)
     // );
     private function post($uri, $payload)
     {
+        $payload['fraud_details'] = array('customer_ip' => $_SERVER['REMOTE_ADDR']);
```

Comment:
@Dinwy Can we also pass the following,
`customer_email`, `browser_language`, and `browser_user_agent`

https://doc.komoju.com/docs/fraud-details

Reply:
I've add more properties in this [commit](https://github.com/degica/komoju-woocommerce/pull/111/commits/2ed40ce19fd213e7aa5603702afab450dabe4794)

Do we need other properties like
'customer_id', 'browser_session_id', 'phone'?

Reply:
@Dinwy Not needed, thank you